http_trace.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 API7.ai, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// TraceSeries contains a series of events (ordered by their happening time).
+type TraceSeries struct {
```

Comment:
I think it is better to add an `id` field to this structure to distinguish these events in the context of concurrent requests.

Reply:
Great. I'll use the snowflake to add an ID.